frontend/public/components/utils/_copy-to-clipboard.scss

```diff
@@ -0,0 +1,14 @@
+.co-copy-to-clipboard {
```

Comment:
The CSS is still not really following BEM. I'd expect rules like

```css
.co-copy-to-clipboard { /* ... */ }
.co-copy-to-clipboard__btn { /* ... */ }
.co-copy-to-clipboard__text { /* ... */ }
```

I'd suggest reading more about it here: http://getbem.com/introduction/